moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -288,10 +291,11 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     {
       if (execute_service_.exists())
       {
-        ROS_WARN_NAMED("move_group_interface",
-                       "\nDeprecation warning: Trajectory execution service is deprecated (was replaced by an action)."
-                       "\nReplace 'MoveGroupExecuteService' with 'MoveGroupExecuteTrajectoryAction' in "
-                       "move_group.launch");
+        ROS_WARN_NAMED("move_group_interface", "\nDeprecation warning: Trajectory execution service is deprecated (was "
+                                               "replaced by an action)."
+                                               "\nReplace 'MoveGroupExecuteService' with "
+                                               "'MoveGroupExecuteTrajectoryAction' in "
+                                               "move_group.launch");
```

Comment:
are you using a different version of clang-format? these changes should not happen